### DISCUSSION TITLE
About line spacing

### DISCUSSION THREAD

**Discussion by a123likeyou** [ImageMagick/ImageMagick#3514] [Help]
I would like to ask how your line spacing is calculated. In my example, the line spacing is relatively small, so no change can be seen. Do you calculate the line spacing based on the font size? If there are 2 lines of 18px font, the line spacing is 10How do you calculate the height of your text? 182+180.1=37.8px? Thanks for your answers

**Comment by snibgo** [comment]
I don't understand the question. Perhaps you can show example commands and images.

See also the documentation at https://legacy.imagemagick.org/Usage/text/, especially https://legacy.imagemagick.org/Usage/text/#attributes and https://legacy.imagemagick.org/Usage/text/#font_info

**Reply by a123likeyou** [reply]
Thank you for your answer, the text seems to have not changed because the line spacing is too small before, but I have solved this problem

**Comment by fmw42** [comment]
What is the command used to create your text that makes you concerned about line spacing?

**Reply by a123likeyou** [reply]
Thank you for your answer, the text seems to have not changed because the line spacing is too small before, but I have solved this problem
